### PR TITLE
Issue 73 filtersortstatus

### LINKED ISSUE #73: Add UI element that indicates filter/sort status
UI should clearly display whether a filter/sort is put in place, and what is the variable that is being filter for/sorted by.

The corresponding UI element should disappear when the _filter default_ or _sort default_ command is called

### PR DESCRIPTION
**Pull request by s-kybound**
Resolves #73

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FilterCommand.java [hunk 1]
```diff
@@ -38,15 +38,37 @@ public class FilterCommand extends InternshipCommand {
...
+        this.filterValue = "";
+    }
+
+    /**
+     * Constructs a Filter Command.
+     * @param filterParameter
+     * @param filterValue
+     * @param predicate
+     */
+    public FilterCommand(String filterParameter, String filterValue, Predicate<Internship> predicate) {
```
Comment (shashahchk) [inline src/main/java/seedu/address/logic/commands/FilterCommand.java:60]: I like that you make the constructor accept the filterParameter and filterValue as well as predicate

File: src/main/java/seedu/address/ui/PredicateComparatorDisplay.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    @FXML
+    private TextArea filterDisplay;
+
+    /**
+     * Initialises the Predicate Comparator Display with the relevant initial information.
+     */
+    public PredicateComparatorDisplay() {
+        super(FXML);
+        // Initialise the displays for sort and filter information.
+        comparatorDisplay.setText("Sorting by c/ in ASC order");
```
Comment (shashahchk) [inline src/main/java/seedu/address/ui/PredicateComparatorDisplay.java:31]: Perhaps standardize whether to put "."

**Review by pygmychameleon — CHANGES_REQUESTED** [review]
Overall LGTM. A few nits

**Review by yongning0310 — APPROVED** [review]
LGTM

**Review by pygmychameleon — APPROVED** [review]
LGTM. Great effort!

**Pull request merged**